src/NServiceBus.MessagingBridge/EndpointRegistry.cs

```diff
@@ -81,12 +81,11 @@ public string GetEndpointAddress(string endpointName)
 
     static string GetNearestCaseInsensitiveMatch(string sourceEndpointName, IEnumerable<string> items)
```

Comment:
Consider changing method name to make it clear that the result is only informational, or alternatively add `defaultValue` or `noEndpointsValue` as an additional parameter.
```suggestion
    static string GetClosestMatchForExceptionMessage(string sourceEndpointName, IEnumerable<string> items)
```